src/python/pants/option/global_options.py

```diff
@@ -1472,9 +1487,16 @@ class BootstrapOptions:
     )
     remote_cache_read_timeout_millis = IntOption(
         advanced=True,
-        default=DEFAULT_EXECUTION_OPTIONS.remote_cache_read_timeout_millis,
+        default=DEFAULT_EXECUTION_OPTIONS.remote_cache_rpc_timeout_millis,
+        removal_version="2.19.0.dev0",
+        removal_hint="Use the `remote_cache_rpc_timeout_millis` option instead.",
         help="Timeout value for remote cache lookups in milliseconds.",
     )
+    remote_cache_rpc_timeout_millis = IntOption(
```

Comment:
Would it make sense to introduce a `DurationOption` type?

Reply:
Which could take human-readable duration strings and convert them internally to a common format.

Reply:
Almost certainly, yea.

Reply:
Do you mind opening a ticket about that?

Reply:
https://github.com/pantsbuild/pants/issues/18698